BridgeAppSDK/Localization.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Localization.swift
+//  BridgeAppSDK
+//
+//  Created by Shannon Young on 2/17/16.
+//  Copyright © 2016 Sage Bionetworks. All rights reserved.
+//
+
+import UIKit
+
+class Localization: NSObject {
+    
+    static var localeBundle = {
+        return NSBundle.init(forClass: Localization.classForCoder())
+    }()
+    
+    static func localizedString(key: String) -> String {
+        return NSLocalizedString(key, tableName: nil, bundle: localeBundle, value: key, comment: "")
+    }
+    
+    // List of the strings used in this module. This func is never called but is included to allow running 
```

Comment:
I see how this dramatically cleans up the localization code in the rest of the app, but I worry about maintaining it. Is there some way to automatically check that it's complete and consistent, i.e. we haven't added or changed any strings elsewhere and forgotten to update here too?